blocks/recommended-articles/recommended-articles.css

```
@@ -1,138 +1,144 @@
 main .recommended-articles-container,
 main .recommended-articles-small-container {
-  background: var(--color-gray-200);
-  padding-bottom: 1rem;
+    background: var(--color-gray-200);
+    padding-bottom: 1rem;
 }
 
 main .recommended-articles-container.appear,
 main .recommended-articles-small-container.appear {
-  visibility: unset;
+    visibility: unset;
 }
 
-main .recommended-articles-container > div {
-  max-width: 100%;
-  text-align: center;
-  padding-top: 16px;
+main .recommended-articles-container>div {
+    max-width: 100%;
+    text-align: center;
+    padding-top: 16px;
 }
 
 main .recommended-articles .article-cards {
-  margin: 0 auto;
-  display: flex;
-  flex-wrap: wrap;
-  padding-bottom: 0;
-  justify-content: center;
-  width: 100%;
+    margin: 0 auto;
+    display: flex;
+    flex-wrap: wrap;
+    padding-bottom: 0;
+    justify-content: center;
+    width: 100%;
 }
 
 @media (min-width: 600px) {
-  main .recommended-articles .article-cards {
-    padding-left: 0;
-    padding-right: 0;
-  }
+    main .recommended-articles .article-cards {
+        padding-left: 0;
+        padding-right: 0;
+    }
 }
 
+
 /* small specific */
+
 main div.recommended-articles-small-content-wrapper {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  align-items: center;
-  max-width: 1200px;
-  margin: 0 auto;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+main div.recommended-articles-small-content-wrapper div.region.container {
+    margin: 0;
```

margin: 0 is a bit aggressive.
I would make the margin-left: 0 only to cancel the auto. (so it would go left aligned.)
```suggestion
    margin-left: 0;
```